client/verta/tests/test_endpoint/test_endpoint.py

```diff
@@ -198,8 +199,11 @@ def test_form_update_body(self):
             Memory("500Mi"),
         ]
 
-        parameter_json = endpoint._form_update_body(resources, DirectUpdateStrategy(), 0)
-        assert parameter_json == {'build_id': 0, 'resources': {'cpu_millis': 500, 'memory': '500Mi'}, 'strategy': 'rollout'}
+        env_vars = {'CUDA_VISIBLE_DEVICES': "1,2", "VERTA_HOST": "dev.verta.ai"}
```

Comment:
Please write "app", not "dev" for **all** tests.
```suggestion
        env_vars = {'CUDA_VISIBLE_DEVICES': "1,2", "VERTA_HOST": "app.verta.ai"}
```